blueprints/gitlab-ci-phantom/files/gitlab-ci.yml

```diff
@@ -0,0 +1,13 @@
+image: node:latest
+
+before_script:
+  - npm config set spin false
+  - npm install -g bower
+  - npm install -g ember-cli
+  - npm install -g phantomjs-prebuilt
```

Comment:
this block can be compressed to `npm install -g bower phantomjs-prebuilt ember-cli` to reduce the number of network requests for duplicate dependencies on those packages.